packages/tempus/src/Tempus_StepperRKBase.hpp

```diff
@@ -30,8 +31,15 @@ class StepperRKBase : virtual public Tempus::Stepper<Scalar>
 
 public:
 
-  virtual int getStageNumber() const { return stageNumber_; }
+  virtual Teuchos::RCP<const RKButcherTableau<Scalar> > getTableau() const
+  { return tableau_; }
+
+  virtual Scalar getOrder()    const{return getTableau()->order();}
+  virtual Scalar getOrderMin() const{return getTableau()->orderMin();}
+  virtual Scalar getOrderMax() const{return getTableau()->orderMax();}
```

Comment:
type should be `int`

Reply:
@sconde  I have debated this.  Are there time integrators with fractional order?

Reply:
Not by designed. But somes schemes are known to have slighter higher convergence rate in the asymptotic regime for certain classes of problems.

If the idea is to potentially test for these in the future, then `Scalar` is fine by me :)

Reply:
I will get this in the next PR.